.github/workflows/doctests.yml

```diff
@@ -39,4 +46,4 @@ jobs:
 
       - name: Run doctests
         run: |
-          pytest --doctest-modules $(cat utils/documentation_tests.txt) -sv --doctest-continue-on-failure
+          pytest --doctest-modules $(cat utils/documentation_tests.txt) -sv --doctest-continue-on-failure --doctest-glob="*.mdx"
```

Comment:
think this is needed to take new `.mdx` into account

Reply:
Surprising, I would not have thought this would be necessary!

Reply:
No that's normal, the default is `"*.rst"` but we obviously have none now.